### PR TITLE
Added appendix with DHS references and disclaimers

### PR DESCRIPTION
**Pull request by nissimsan**

### CODE REVIEW

**Comment by OR13** [comment]
@mprorock @mkhraisha you will likely need to do the sam, please open issues to track for yourselves, or paste your contract numbers into the suggest changes feature of github.

File: docs/index.html [hunk 1]
```diff
@@ -564,5 +564,17 @@ <h2>Internationalization Considerations</h2>
         take into account when utilizing this data model.
       </p>
     </section>
+
+    <section class="appendix">
+      <h2>Acknowledgements</h2>
+      <p>
+        Portions of the work on this specification have been funded by the
+        United States Department of Homeland Security's (US DHS) Silicon Valley
+        Innovation Program under contracts 70RSAT20T00000003, and
```
Comment (mprorock) [inline docs/index.html:573]:
```suggestion
        Innovation Program under contracts 70RSAT20T00000044, 70RSAT20T00000003, and
```

**Review by mprorock — CHANGES_REQUESTED** [review]
our OTA# 70RSAT20T00000044 is suggested as an inline change if you don't mind merging that in as well to prevent multiple PRs

File: docs/index.html (hunk 1 above)
Comment (mkhraisha) [inline docs/index.html:573]:
```suggestion
        Innovation Program under contracts  70RSAT20T00000003, 70RSAT20T00000031,70RSAT20T00000043, and
```

Comment (OR13) [reply docs/index.html]: @mkhraisha I merging @mprorock 's suggestion busted this, can you do it again?

**Comment by OR13** [comment]
I would like to incrementally merge things here, folks should be responsible for PRs adding contract numbers they have, there is only so much of that we should solo... there is also the contribution graph to consider...

File: docs/index.html [hunk 2]
```diff
@@ -564,5 +564,17 @@ <h2>Internationalization Considerations</h2>
...
       </p>
     </section>
+
+    <section class="appendix">
+      <h2>Acknowledgements</h2>
+      <p>
+        Portions of the work on this specification have been funded by the
+        United States Department of Homeland Security's (US DHS) Silicon Valley
+        Innovation Program under contracts 70RSAT20T00000044, 70RSAT20T00000003, and
+        70RSAT20T00000033. The content of this specification does not
```
Comment (TallTed) [inline docs/index.html:574]:
```suggestion
        Innovation Program under contracts
        70RSAT20T00000003,
        70RSAT20T00000033,
        and
        70RSAT20T00000044.
        The content of this specification does not
```
Keeping the contracts in alphanumerical order and stacked as above in the HTML source makes it easier to check whether one is already listed, avoiding duplication, and speeding edits when needed.

**Comment by OR13** [comment]
@mprorock @nissimsan can you review this again, I have accepted changes.

**Review by OR13 — APPROVED** [review]

File: docs/README.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 This directory is used to build and serve the content hosted at:
 
-- https://w3id.org/traceability
+- https://w3id.org/traceability-vocab
```
Comment (OR13) [inline docs/README.md:3]:
```suggestion
- https://w3id.org/traceability
```

Comment (OR13) [reply docs/README.md:3]: This change broke a link.

**Review by OR13 — CHANGES_REQUESTED** [review]
Please accept the link suggestion.

File: docs/index.html [hunk 4]
```diff
@@ -464,7 +464,7 @@ <h5>E-Commerce</h5>
           </dl>
         </section>
       </section>
- 
+
```
Comment (OR13) [inline docs/index.html:467]:
```suggestion
```

**Review by OR13 — APPROVED** [review]

**Review by mprorock — APPROVED** [review]

**Pull request merged**